erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -44,17 +44,18 @@ def before_submit(self):
 	def validate_delivery_date(self):
 		order_delivery_dates = [frappe.db.get_value("Sales Order Item", location.get("sales_order_item"), "delivery_date")
 			for location in self.locations if location.get("sales_order_item")]
-		self.delivery_date = min(order_delivery_dates)
+		if order_delivery_dates:
+			self.delivery_date = min(order_delivery_dates)
 
 	def validate_stock_qty(self):
 		"""User should not allowed to create pick list if sales order item qty exceed."""
 		for item in self.locations:
 			if item.get("sales_order_item"):
 				ordered_item_qty = frappe.db.get_value("Sales Order Item", item.get("sales_order_item"), "qty")
 				prev_picked_qty = frappe.get_all("Pick List Item", filters={"sales_order_item":item.get("sales_order_item"), "docstatus":1}, fields=['sum(qty) as prev_picked_qty'])
+				prev_picked_qty = prev_picked_qty[0].prev_picked_qty
```

Comment:
Keep this on line 57, instead do:

```suggestion
				prev_picked_qty = flt(prev_picked_qty[0].prev_picked_qty)
```